src/components/ContextualSaveBar/tests/ContextualSaveBar.test.tsx

```diff
@@ -12,58 +12,111 @@ describe('<ContextualSaveBar />', () => {
   };
 
   it('calls the contextual save bar on mount with correct values', () => {
-    const {frame} = mountWithContext(<ContextualSaveBar {...props} />);
-    expect(frame.setContextualSaveBar).toHaveBeenCalledWith({
+    const mockFrameContext = {
+      frame: {
+        setContextualSaveBar: jest.fn(),
+        removeContextualSaveBar: jest.fn(),
+        showToast: noop,
+        hideToast: noop,
+        startLoading: noop,
+        stopLoading: noop,
+      },
+    };
+
+    mountWithAppProvider(
+      <Provider value={mockFrameContext}>
+        <ContextualSaveBar {...props} />
```

Comment:
I currently get a TypeScript error on `ContextualSaveBar` saying:

```
Type '{ saveAction: { content: string; onAction: () => void; }; discardAction: { content: string; onAction: () => void; }; message: string; }' is missing the following properties from type 'Readonly<Props & WithAppProviderProps & WithContextTypes<FrameContext>>': polaris, context
```

The code still compiles properly, and tests pass, but this feels wrong to me, and maybe related to the issue in `ContextualSaveBar` where I have to pull `polaris`, and `context` off `this.props`.

Reply:
Looks like this was a result of using the wrong version of typescript (`3.2.2`, instead of `3.1.6`).